extensions/stackdriver/stackdriver.cc

```diff
@@ -189,6 +189,12 @@ bool StackdriverRootContext::onConfigure(
 
   node_info_cache_.setMaxCacheSize(config_.max_peer_cache_size());
 
+  // All the necessary reporting objects should be initialized.
```

Comment:
This needs to move to the top of `onConfigure`.

Reply:
It should match the `release-1.5` changes:

https://github.com/istio/proxy/blob/master/extensions/stackdriver/stackdriver.cc#L140-L146.

And we should probably match `master`.

Reply:
OK. we are going to update 1.5 to match this placement.